src/python/pants/engine/README.md

```diff
@@ -136,27 +145,24 @@ def concat(files):
   yield ConcattedFiles(''.join(fc.content for fc in file_content_list))
 ```
 
-This `@rule` declares that: "for any Subject for which we can compute `Files`, we can also compute
-`ConcattedFiles`". Each yielded `Get` request results in FileContent for a different File Subject
-from the Files list.
+This `@rule` declares that: "for any `Params` for which we can compute `Files`, we can also compute
+`ConcattedFiles`". Each yielded `Get` request results in FileContent for a different File `Param`
+from the Files list. And, happily, all of these requests will proceed in parallel.
 
-### Variants
+### Advanced Param Usage
```

Comment:
I'm not sure if this new title is fitting. I expected it to be about _multiple_ advanced ways I can use Params after I figure out the basic usage. Instead, this seems to be about how rules can expect the same Param and still resolve to a different rule? I'm not sure what a better title would be for that.. suggestions from others?

Reply:
So, I think that one of the issues is that drafting an example for this section is blocked on #7490: currently the only way to inject more than one parameter at a time is at the root of a request.